portfolio/src/main/webapp/blog.html

```diff
@@ -4,15 +4,16 @@
     <meta charset="UTF-8">
     <title>Kayla Leung</title>
 
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="blogStyle.css">
     <link rel="stylesheet" href="font.css">
 
+    <script src="script.js"></script>
     <script src="blogScript.js"></script>
     <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     
   </head>
-  <body>
+  <body onload="getComments()">
 
     <!-- Toggle Side View -->
     <input type="checkbox" name="" \>
```

Comment:
The slash is backwards (should be a forward slash), but it's not necessary and the Google HTML style guide recommends leaving it out. I think it's clear without too.